project/src/main/test/java/com/google/sps/test/PlaceTest.java

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * The package that currently holds all the test files of the food-mood project.
+ * @since 1.0
+ * @author karenav
+ * @version 1.1
+ */
+package com.google.sps.test;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.data.Place;
+
+@RunWith(JUnit4.class)
+public final class PlaceTest {
+
+  /** A stub Place name. */
+  private static final String VALID_NAME = "name";
+  /** A stub Place website URL. */
+  private static final String VALID_WEBSITE = "website@google.com";
+  /** A stub Place phone number. */
+  private static final String VALID_PHONE = "+97250-0000-000";
+  /** A valid Place rating. */
+  private static final int VALID_RATING = 4;
+  /** A valid Place price level. */
+  private static final int VALID_PRICE_LEVEL = 3;
+  /** A stub Place longitude. */
+  private static final long VALID_LONGITUDE = 35.35;
+  /** A stub Place latitude. */
+  private static final long VALID_LATITUDE = 30.30;
+
+  @Test
+  public void create_invalidLowRating_throwsIllegalArgumentException() {
+    final int invalidLowRating = 0;
```

Comment:
Remove final, same for the rest.

Reply:
done

Reply:
when I remove it, I get an error that these are magic numbers..
why should the final be removed? from what I spoke to Bar it was good practice to declare final when the var won't change in the function